integration/data/snapshot_tree.py

```diff
@@ -63,55 +63,77 @@ def snapshot_tree_verify_relationship(address, snap, strict):
     info = cmd.snapshot_info(address)
 
     assert snap["0a"] in info
-    assert snap["0b"] in info[snap["0a"]]["children"]
+    assert snap["0b"] in \
```

Comment:
We no longer need to change those, right?

Reply:
fixed